src/Components/Toolbar/SaveCharacter/SaveAlerts/displayAlert.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import { useAppSelector } from 'features/reduxHooks';
+import { Alert } from 'react-bootstrap';
+
+export const DisplayAlert: React.FC<{ alertType: string }> = ({ alertType }) => {
+  const characterName = useAppSelector((state) => state.character.name);
+  const nameText = characterName;
+  const [show, setShow] = useState(true);
+
+  const createAlert = (variant: string, text: string) => {
+    if (show) {
+      return (
+        <div className="save-alert">
+          <Alert variant={variant} onClose={() => setShow(false)} dismissible>
+            <Alert.Heading>{text}</Alert.Heading>
+            <div className="d-flex justify-content-end"></div>
+          </Alert>
+        </div>
+      );
+    } else return;
```

Comment:
you've got a lot of returns that aren't needed. i think here you wouldn't need the else return. there's quite a few others if you just search `return;` maybe go through them and delete them. most of them from what I can you won't need. unless you need to terminate the function early to stop it executing code further down below the empty return then you shouldn't need them
```ts
// not needed
if(friend){
 console.log('hey mate')
}
return
```
```ts
// needed
if(friend){
 console.log('hey mate')
}
return
console.log('ya mums a 🐷') // wouldn't be executed if friend is true, but a biff would be activated if !friend
```